### PR TITLE
feat: add lnurl-withdraw support

### PR DESCRIPTION
**Pull request by dylancom**
Tested with Kollider: https://pro.kollider.xyz/ & https://stacker.news/

### CODE REVIEW

**Comment by kiwiidb** [comment]
Is this PR still necessary? I can already withdraw from Kollider and Stacker News, or is that using webLN?

**Comment by dylancom** [comment]
At Kollider, from the wallet page it's using WebLN. Try to withdraw in the bottom left @ https://pro.kollider.xyz/

File: src/app/screens/LNURLWithdraw.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
+import { useState, MouseEvent } from "react";
+import axios from "axios";
+
+import { LNURLWithdrawServiceResponse } from "../../types";
```
Comment (escapedcat) [inline src/app/screens/LNURLWithdraw.tsx:4]: This is only used as a type, right? Can `import type ...` be used?

Comment (dylancom) [reply src/app/screens/LNURLWithdraw.tsx]: Correct!

**Comment by bitkarrot** [comment]
Any ETA on when this pull request will get merged? I'd like to make use of the feature. Thanks much in advance! Thanks @dylancom @escapedcat

**Review by bumi — APPROVED** [review]
utACK

**Pull request merged**